content/posts/why-sre/index.mdx

```diff
@@ -0,0 +1,159 @@
+It was established out of frustration with not being able to move fast enough.
+The time it took to provision clusters for customers limited how fast they could grow.
+The ops team at the time spent a lot of time 'hands on' with clusters, managing them and dealing with escalations.
+The concept of SRE was discussed as a way to reorganise and think about the problem differently.
+
+It started with some reasonable goals e.g. Can we halve the time it takes to create a cluster, thereby allowing double the amount per year?
+As that goal was reached, it got revised again and again until it got to the point where it didn't matter how many clusters needed to be created.
+The SRE team had sufficiently automated away so much of the manual processes around creation that it was a non-event for a new cluster to be created.
+
+Having an initial goal to rally around can help when forming a new team or making a transition.
+There are many other aspects that can boost the effort as well.
+Hiring software engineers who have a 'systems mindset' and can think about scale was important here.
+A lot of focus was put on the processes and people when hiring and building the team rather than the technology.
+Embracing risk and removing any fear of retribution was important to establish early on.
+As was having a foundation of trust and building on it.
+In general, focus was on the people, the practice, education, training and enablement.
+
+You can read more about this transition in this blog post from 2020, [From Ops to SRE: Evolution of the OpenShift Dedicated Team][2]
+
+## Case Study 2: Ansible Automation Platform on Azure
+
+The team behind the Ansible Automation Platform is a relatively new team.
+It is an ensemble team made up of developers and SREs.
+The SRE team's job is to ensure the service deploys, stays running and is kept up to date.
+It runs as a Managed Application on Azure, which was a major reason for establishing a new SRE team.
+There are other 'centralised' SRE teams at Red Hat that enable services running on OpenShift.
+However, this team needed to specialise in Ansible Automation Platform as well as Azure.
+The reasons for 'SRE' specifically were twofold.
+It was already an established practice in Red Hat with various practicing teams to draw knowledge from.
+Also, the management team had previous experience with many of the SRE principles and practices.
+
+One the biggest motivating factors for this team is pain.
+Both customer pain and the team's pain.
+An example of customer pain would be not being able to purchase and deploy at the press of a button.
+The team saw this from the customers perspective and looked at ways of automating away the manual steps and engagements.
+They identified areas where a deploy could fail late in the process and someone would have to manually reset and trigger things again.
+The solution here was adding pre-installation checks.
+It's a reactive model, but it's also actively managing TOIL early on.
```

Comment:
```suggestion
It's a reactive model, but it's also [actively managing TOIL early on](https://github.com/operate-first/sre/blob/main/sre_maturity.md#toil-management-and-operational-improvements).
```

This link will work and have proper content once https://github.com/operate-first/sre/pull/21/files is finished (today) and merged.